### DISCUSSION TITLE
Can't install Windows 11 x86 - This PC can't run  Windows 11

### DISCUSSION THREAD

**Discussion by nikolan79** [utmapp/UTM#4804] [UTM Help (macOS)]
Hello,

unfortunately I can't install Windows 11. I already tried all the solutions modifying the registry but without any success. The following values have been set in the LabConfig key.

BypassCPUCheck = 1
BypassRAMCheck = 1
BypassSecureBootCheck = 1
BypassTPMCheck = 1
BypassStorageCheck = 1

I also followed the steps on the UTM website. So I guess everything should be setup correctly. 

My system is as followed:

Apple M1 Pro

VM Config:
Architecture: x86_64
System: Standard PC (Q35 + ICH9, 2009) (alias of pc-q35-7.0) (q35)
Memory: 8 GB
Storage: 64 GB

Can some help me?

**Comment by stossy11** [comment]
You could use unofficial "Distros" of windows for example tiny11 or Windows 11 Lite they disable TPM and the min ram is 256MB and it worked for me.

**Comment by MaffooClock** [comment]
I'm having this same issue (VM configured the same as OP), except on a M2.  [The documentation](https://docs.getutm.app/guides/windows/#this-pc-cant-run-windows-11) guides you through modifying the registry, but the `LabConfig` key and values it tells me to create already exist (and are set to 1), yet the error persisted.

### Solution:
I found that if I changed the "CPU Cores" option from "Default" to "2", Windows Installer worked fine, and I didn't need to hack the registry.

**Comment by wrmack** [comment]
I  suspect it is because you trying to install x86 on an M1.  You need ARM64 on an M1. A Windows 11 ARM64 will run x64 apps OK. If interested, I have set out the steps I took to install Windows 11 here: https://gist.github.com/wrmack/29d580a0bd665bace21609d5ef675ba1

**Comment by MaffooClock** [comment]
> You need ARM64 on an M1

Nope -- that's one of the advantages of using UTM, that you can create VMs that don't match the architecture of the host.

In this case, the "This PC can't run Windows 11" error isn't caused by lack of TPM or Secure Boot or whatever (because the registry hack is supposed to take care of that); instead, there must be two or more CPU cores, and evidently a registry hack can't bypass this.
